advanced/events.md

```diff
@@ -150,6 +150,8 @@ The following events are provided by the {{ site.productname }} editor.
 | ResizeContent          | (Same data as the native [resize event](https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event))          | (iframe mode only) Fired when the iframe window is resized.                                                                                                                |
 | Load                   | N/A                                                                                                                     | (iframe mode only) Fired when the editor iframe content has finished loading.                                                                                              |
 | ResolveName            | `{ name: string, target: Element }`                                                                                     | (iframe mode only) Fired when the editor tries to resolve the name of an element to render the status bar path.                                                            |
+| BeforeOpenNotification | `{ notification: NotificationSpec }`                                                                                    | Fired before a notification is displayed. For information on the Notifications Specification (`NotificationSpec`), see: [Create custom notifications]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                       |
+| OpenNotification       | `{ notification: NotificationSpec }`                                                                                    | Fired after a notification is displayed. For information on the Notifications Specification (`NotificationSpec`), see: [Create custom notifications]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                        |
```

Comment:
I just realized this is incorrect sorry. This gets passed the notification API, not the notification spec.

Reply:
To clarify further, `BeforeOpenNotification` gets passes a `NotificationSpec`, whereas `OpenNotification` gets passed a `NotificationApi` instance.

Reply:
```suggestion
| OpenNotification       | `{ notification: NotificationApi }`                                                                                    | Fired after a notification is displayed. For information on the Notifications API (`NotificationApi`), see: [Create custom notifications]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                        |
```